WordPress/Tests/NamingConventions/ValidVariableNameUnitTest.inc

```diff
@@ -0,0 +1,91 @@
+<?php
+$varName  = 'hello';
```

Comment:
Could you add `// OK` or `// Bad` comments to each line so that it is more obvious what is supposed to fail and what is ok?